<?xml version="1.0"?>

<launch>
  <!--node pkg="torob" type="viewer" name="viewer1" output="screen" >
    <param name="vmap_topic"      value="/tb1/vmap"/>
  </node-->

  <node pkg="torob" type="viewer" name="viewer">
    <param name="vmap_topic"      value="/tb/vmap"/>
    <param name="only_obstacle"      value="true"/>
  </node>

  <node pkg="torob" type="viewer" name="viewer2">
    <param name="vmap_topic"      value="/tb/scan_vmap"/>
  </node>

  <!--node pkg="torob" type="viewer" name="viewer3" output="screen" >
    <param name="vmap_topic"      value="/tb3/vmap"/>
  </node-->

  <node pkg="torob" type="net_receive" name="net_receive" output="screen" >
    <param name="port"      value="1414"/>
    <param name="ns_topic"  value="/tb"/>
  </node>

</launch>
